### ISSUE TITLE
Ignored AR-Scale in iOS

### ISSUE THREAD

**Issue by llllkatjallll** [needle-tools/needle-engine-support#160]
### Describe the bug 💬

I added a "USDZ Exporter" component in my scene to be able to view the scene in AR on iOS. Insite this component I dragged the object/scene I want to export into the field "Object to export".
Additionally I have a "Web AR Session Root" component in the scene with the Ar-Scale set to 25.
<img width="417" alt="20230704-122437-Unity" src="https://github.com/needle-tools/needle-engine-support/assets/38395689/e6ca7842-70c4-4416-919d-e7e5038f26d5">

**My expectation:**
The scene will be exported for iOS with the same scale as it does for Android (scale 25). 

**What occurred:**
In AR-mode in iOS the scene has the scale 1.

**Workaround**
If I leave the field "Object to export" empty, my scene has the right scale in AR in iOS.

### Operating System 👩‍💻

Windows

### What browsers are you seeing the problem on? 🏄‍♂️

Safari

### Editor Version 🎲

2021.3

### Needle Exporter version

3.6.15

### Steps to reproduce 🔢

```bash
1.Get sure you have "USDZ Exporter" and  "Web AR session root" (with set ar scale (e.g. 25)) in your project. 
3. Make the Object you want to export really big! Big like a house! :house_with_garden:
2. Drag the Object you want to see in AR in the field of USDZ Exporter "Object to export"
3. Test the project on iOS in AR. The object is still house-big and not 25-times smaller.
```

### Validations 🩹

- [X] Follow our [Code of Conduct](https://github.com/needle-tools/needle-engine-support/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://docs.needle.tools).
- [X] Check that there isn't [already an issue](https://github.com/needle-tools/needle-engine-support/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Needle-Engine-specific issue. For example if it's a Unity related bug, it should be reported to [Unity](https://unity3d.com/unity/qa/bug-reporting) instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/needle-tools/needle-engine-support/discussions) or join our [Discord Chat Server](https://discord.needle.tools).
- [ ] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by marwie** [comment]
should be fixed in the next release > 3.7.5-alpha

**Comment by llllkatjallll** [comment]
thank you!
